emsdk.py

```diff
@@ -1332,6 +1332,16 @@ def emscripten_npm_install(tool, directory):
       return False
 
   print('Done running: npm ci')
+
+  try:
+    subprocess.check_output([sys.executable, 'bootstrap.py'],
+      cwd=directory, stderr=subprocess.STDOUT, env=env,
+      universal_newlines=True)
+  except subprocess.CalledProcessError as e:
+    errlog('Error running %s:\n%s' % (e.cmd, e.output))
+    return False
```

Comment:
I don't think the try/catch here should be necessary.

`check_output`, on its own, should already report the name of the script that failed (I think).

Alternatively, perhaps this try/catch with `check_output` can be factored out since its a pattern used above.

lgtm either way though.

Its a shame we can't really a CI test for this because I think it requires compiling llvm and binaryen from source, right?

Reply:
The idea here I think is to stop propagating the exception - I reused the same pattern that the above existing try-catch uses.

Yeah, there is probably no good way to avoid testing this unless we would compile from source. (we could inject some kind of fake mocking, but that would have some brittle)